app/src/main/java/bean/pwr/imskamieskiego/utils/EventWrapper.java

```diff
@@ -0,0 +1,41 @@
+package bean.pwr.imskamieskiego.utils;
+
+public class EventWrapper<T> {
+
+    private boolean isHandled;
+    private T data;
+
+    /**
+     * Create new event wrapper for data. Wrapper preserve information, was data read before.
```

Comment:
Nie wiem czy o to chodziło czy coś zgubiłeś
Create new event wrapper for data. Wrapper preserves information, "if" data was read before.